lib/controllers/widget/device_list.dart

```diff
@@ -145,20 +193,18 @@ class _DeviceListState extends State<DeviceList> {
       appBar: AppBar(
         title: Text('Chrome OS Devices'),
       ),
-      body: _loading == true && _list == null
-          ?
-          // Loading screen if its the first loading of the list
-          Container(
-              child: Center(
-                child: Text('Loading...'),
-              ),
-            )
-          : Container(
-              child: _errorOnLoading == true
-                  ?
-                  // Warning notification when a error on loading ocurrs
-                  Center(child: Text('$_warning'))
-                  : _list.chromeosdevices.length == 0
+      body: _errorOnLoading
+          ? _warning.statusCode > 499
+              ? _alertAndRetry(_warning)
+              : _alertAndLogIn(_warning, false)
```

Comment:
The false should be conditional, we should send true if user just logged in and got a token, otherwise false.

Reply:
Or just remove the bool, since it's always false.

Reply:
Already fixed the condition.